about.css

```diff
@@ -0,0 +1,534 @@
+@import url(//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE);
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+:root {
+  --primary_1: #000;
+  --primary_2: #ec5242;
+  --primary_3: #78829b;
+  --primary_4: #2e323b;
+  --primary_5: #cac6c6;
+  --secondary_1: #fff;
+  --secondary_2: #ece9e9;
+  --secondary_3: #d1cfcf;
+  --secondary_4: #d3d3d3;
+  --secondary_5: #f5220a;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif;
+  overflow-x: hidden;
+}
+
+.header-menu,
+.desktop-menu {
+  display: none;
+}
+
+.header-menu {
+  display: none;
+}
+
+.sidebar {
+  display: none;
+}
+
+#about-handburger {
+  padding-top: 6%;
+  margin: 0 1em;
+}
+
+.handburger {
+  content: "\f0c9";
+  font-size: 40px;
+  color: var(--primary_4);
+}
+
+.mobile-menu {
+  display: none;
+  background-color: #3c3a39;
+  width: 100%;
+  margin-top: 0;
+  height: 100%;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  position: fixed;
+  list-style: none;
+}
+
+.menu1 {
+  margin-top: 30px;
+  margin-right: 10px;
+}
+
+.mobile-menu .sidebar-cancel {
+  margin: 30px;
+  color: var(--secondary_1);
+}
+
+.mobile-about-footer a {
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: bolder;
+  margin-left: 20px;
+}
+
+.mobile-menu .menuPopUp a {
+  display: flex;
+  flex-direction: column;
+  color: var(--secondary_1);
+  margin-top: 3rem;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  list-style: none;
+  margin-right: 2rem;
+  text-align: center;
+  text-decoration: none;
+}
+
+.about-first-section {
+  background-color: var(--primary_5);
+  margin-bottom: 50px;
+}
+
+#about-first-section {
+  margin-right: 1em;
+  margin-left: 1em;
+  background-image: url('../images/bg-decorator-1.png'), url('../images/bg-decorator-2.png'), url('../images/bg-decorator-3.png'), url('../images/bg-decorator-4.png');
+  background-repeat: no-repeat;
+  background-size: 20%, 24%, 10%, 10%;
+  background-position: 0 10%, 90% 100%, 80% 50%, 90% 20%;
+}
+
+.first-section-background {
+  background-color: var(--secondary_1);
+  opacity: 0.8;
+}
+
+.about-mobile-first-section h1 {
+  font-family: 'Lato', sans-serif;
+  color: var(--primary_2);
+  font-size: 32px;
+  padding-top: 6%;
+  width: 100%;
+  text-align: center;
+}
+
+.about-mobile-first-section h2 {
+  text-align: center;
+  font-family: 'COCOGOOSE', Arial, sans-serif;
+  font-size: 43px;
+  width: 100%;
+  color: var(--secondary_5);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.about-mobile-first-section p {
+  margin-top: 15%;
+  background-color: var(--secondary_1);
+  border: 2px solid var(--secondary_4);
+  color: var(--primary_4);
+  padding: 20px;
+  line-height: 20px;
+  width: 100%;
+  text-align: center;
+}
```

Comment:
- [ ] Please add margin around the **"An Javascript community that believes in the value of openness..."** paragraph and make sure it's not overflowing or collapsing with the sides. You can use the CSS `margin` for adding margin around the paragraph box and to avoid overflowing of the content I would suggest you to remove `width:100%`.

- [ ] Make sure that the main heading's font size is appropriate and your it fits the first two lines just like in the provided design. (Note: if your heading is long just adjust the font-size to make the heading fit in two lines)
- [ ] Please remove grey `background-color` from `.about-first-section` as it's not looking good and making your content look cramped.

Cindy's design  |Your design
----|----
<img width="359" alt="Screenshot 2023-07-07 at 6 55 08 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/1b8317fc-8907-4f46-b727-ce194dc07460">|<img width="319" alt="Screenshot 2023-07-07 at 6 57 47 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/3c3512af-dca6-4fd7-a844-0508e8ab185d">